### DISCUSSION TITLE
slice in open5gs

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [herlesupreeth/docker_open5gs#339] [General]
Hi @herlesupreeth 

How many slice docker open5gs will support .

is possible to add multi slice ?

**Comment by herlesupreeth** [comment]
You can have as many slices as open5gs supports.

**Reply by Sivanesh1992** [reply]
Thankyou @herlesupreeth  what are the changes i need to do yaml if i need to add slices .

**Reply by herlesupreeth** [reply]
you could take a look at this commit - https://github.com/herlesupreeth/docker_open5gs/commit/ca0d937688c302c559a29cdfc5f4a61ef49cb335 in the branch slicing

**Reply by Sivanesh1992** [reply]
Thankyou @herlesupreeth
